test/Traveler-test.js

```diff
@@ -5,16 +5,21 @@ import travelersData from './travelers-dum.js';
 import tripsData from './trips-dum.js';
 import destinationsData from './destinations-dum.js';
 import Traveler from '../src/Traveler.js';
+import Trip from '../src/Trip.js';
 
 let traveler1;
 let allTrips1;
 let pastTrips1;
 let upcomingTrips1;
+let trips;
 
-describe('Traveler', () => {
+describe.only('Traveler', () => {
 
   beforeEach(() => {
     traveler1 = new Traveler(travelersData[1]);
+    trips = tripsData.map(trip => {
+      return new Trip(trip)
+    })
```

Comment:
I wonder if you truly need to map here since you only need one trip to test for. Will you be testing for different scenarios with multiple trips??